src/commands/releases.rs

```diff
@@ -104,6 +105,17 @@ pub fn make_app<'a, 'b: 'a>(app: App<'a, 'b>) -> App<'a, 'b> {
                         This requires that the command is run from within a git repository.  \
                         sentry-cli will then automatically find remotely configured \
                         repositories and discover commits."))
+            .arg(Arg::with_name("local")
```

Comment:
Should include `.conflicts_with("auto")` and/or `.conflicts_with_all()` for other supporting arguments.

Reply:
Same, but inverse for `auto`

Reply:
I checked clap docs and its says that we do not need to define the conflict for both arguments. Apparently once is sufficient.